tests/controller-test.js

```diff
@@ -445,6 +445,8 @@ OO = {
 
     Html5Skin.toggleMute.call(controllerMock, true);
     Html5Skin.toggleMute.call(controllerMock, false);
+  
+    Html5Skin.moveToDirection.call(controllerMock);
```

Comment:
This file was deprecated after recent updates. Please add regular unit tests in html5skin-test.js